chain/rosetta-rpc/src/models.rs

```diff
@@ -979,6 +979,34 @@ pub(crate) struct TransactionIdentifier {
     pub hash: String,
 }
 
+impl TransactionIdentifier {
+    /// Returns an identifier for a NEAR transaction with given hash.
+    pub(crate) fn transaction(tx_hash: &near_primitives::hash::CryptoHash) -> Self {
+        Self::construct("tx", tx_hash.to_base())
+    }
+
+    /// Returns an identifier for a NEAR receipt with given hash.
+    pub(crate) fn receipt(receipt_hash: &near_primitives::hash::CryptoHash) -> Self {
+        Self::construct("receipt", receipt_hash.to_base())
+    }
+
+    /// Returns an identifier constructed as <prefix>:<hash>.
+    ///
+    /// Note: If constructing identifiers for transactions or receipts, use
+    /// [`transaction`] or [`receipt`] methods instead (this method will give
+    /// the wrong result).
+    pub(crate) fn new(
```

Comment:
This feels off after having `transaction` and `receipt` constructors, and given there is another `construct` method. I would have hard time deciding which method to use and why.

I would:
1. rename `construct` to `from_prefix_and_hash` and make it private
2. rename `new` to `block_event` or something along those lines